### DISCUSSION TITLE
Password not saved - UI changes?

### DISCUSSION THREAD

**Discussion by 1Tomber** [hassio-addons/addon-ssh#639] [Ideas]
Dear community,
I just had trouble installing the addon, it wouldn't save the password in the configuration and always spit out 502 Bad Gateway. I was running circles, looks through the issues.
[https://github.com/hassio-addons/addon-ssh/issues?q=is%3Aissue+gateway+password](https://github.com/hassio-addons/addon-ssh/issues?q=is%3Aissue+gateway+password)

After 15 minutes I found the problem: I was hitting the save button at the end of the configuration page (red circle) and not the one for the first section (blue circle).

![image](https://github.com/hassio-addons/addon-ssh/assets/24652659/c9b3a7a8-2d94-4838-8cfb-ef205baa4b92)

So I raise the question if individual save buttons are really necessary in the scope of the configuration section to avoid further confusion on the user side.

The thing is, I would have communicated my issue the same way as the ones before me, 'I hit save and nothing is saved', just because I overlooked the first save button and headed straight to the end of the page to hit the last button.

**Comment by frenck** [comment]
Thanks for your feedback, but your are reporting it to the wrong project. Add-ons do not manage/create this interface. The Home Assistant project does.

Therefore closing this on this end.

../Frenck
